test/unittests/CMakeLists.txt

```diff
@@ -10,9 +10,10 @@ target_link_libraries(fizzy-unittests PRIVATE fizzy::fizzy-internal fizzy::test-
 target_sources(
     fizzy-unittests PRIVATE
     api_test.cpp
-    bit_cast_test.cpp
     capi_test.cpp
     constexpr_vector_test.cpp
+    cxx20_bit_test.cpp
+    cxx20_span_test.cpp
```

Comment:
@gumb0 @chfast what do you think about this renaming?

Reply:
I think it's fine.